packages/header-checker-lint/src/header-checker-lint.ts

```diff
@@ -74,6 +75,23 @@ class Configuration {
 
 export = (app: Application) => {
   app.on('pull_request', async context => {
+    let remoteConfiguration: ConfigurationOptions = DEFAULT_CONFIGURATION;
+    try {
+      const canditateConfiguration = await context.config(
```

Comment:
Instead of the `if` below to do the case, you should be able to do:
```js
remoteConfiguration = await context.config<ConfigurationOptions>( ...
```

I think you can skip the separate candidateConfiguration thingy
config = config ? 

Then outside of the catch:
```js
remote

Reply:
`context.config()` can return `null` so I think I still need to check the return